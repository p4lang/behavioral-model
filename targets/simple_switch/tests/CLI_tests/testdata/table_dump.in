table_num_entries ecmp_group
act_prof_create_group ecmp_action_profile
act_prof_create_member ecmp_action_profile set_nhop 10.0.0.1 1
table_indirect_add ecmp_group 10.0.0.1/12 => 0
act_prof_add_member_to_group ecmp_action_profile 0 0
table_indirect_add_with_group ecmp_group 10.0.1.1/32 => 0
table_dump ecmp_group
table_num_entries ecmp_group
table_dump_entry ecmp_group 0
act_prof_dump_member ecmp_action_profile 0
act_prof_dump_group ecmp_action_profile 0
table_dump_entry_from_key ecmp_group 10.0.0.1/12
table_clear ecmp_group
table_num_entries ecmp_group
