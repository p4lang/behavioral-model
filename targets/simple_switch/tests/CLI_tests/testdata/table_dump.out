????
0
????
Group has been created with handle 0
????
Member has been created with handle 0
????
Adding entry to indirect match table ecmp_group
Entry has been added with handle 0
????
????
Adding entry to indirect match table ecmp_group
Entry has been added with handle 1
????
==========
TABLE ENTRIES
**********
Dumping entry 0x0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Index: member(0)
**********
Dumping entry 0x1
Match key:
* ipv4.dstAddr        : LPM       0a000101/32
Index: group(0)
==========
MEMBERS
**********
Dumping member 0
Action entry: set_nhop - 0a000001, 01
==========
GROUPS
**********
Dumping group 0
Members: [0]
==========
Dumping default entry
EMPTY
==========
????
2
????
Dumping entry 0x0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Index: member(0)
????
Dumping member 0
Action entry: set_nhop - 0a000001, 01
????
Dumping group 0
Members: [0]
????
Dumping entry 0x0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Index: member(0)
????
